src/Controllers/Metrics/Controller.php

```diff
@@ -84,15 +84,24 @@ public function metrics(): Response
                 'type' => 'gauge',
                 'help' => 'The total number of licenses',
                 ['labels' => ['type' => 'has_car'], 'value' => $this->stats->licenses('has_car')],
-                ['labels' => ['type' => 'forklift'], 'value' => $this->stats->licenses('forklift')],
-                ['labels' => ['type' => 'car'], 'value' => $this->stats->licenses('car')],
-                ['labels' => ['type' => '3.5t'], 'value' => $this->stats->licenses('3.5t')],
-                ['labels' => ['type' => '7.5t'], 'value' => $this->stats->licenses('7.5t')],
-                ['labels' => ['type' => '12t'], 'value' => $this->stats->licenses('12t')],
-                ['labels' => ['type' => 'ifsg_light'], 'value' => $this->stats->licenses('ifsg_light')],
-                ['labels' => ['type' => 'ifsg'], 'value' => $this->stats->licenses('ifsg')],
+                // unconfirmed licenses
+                ['labels' => ['type' => 'forklift_unconfirmed'], 'value' => $this->stats->licenses('forklift')],
```

Comment:
This should just add [additional labels](https://prometheus.io/docs/concepts/data_model/#notation) to allow aggregation / calculation
```suggestion
                ['labels' => ['type' => 'forklift', 'confirmed' => true], 'value' => $this->stats->licenses('forklift')],
```
(same for the following)

Reply:
In addition eventually the confirmed and unconfirmed stats could be shown after each other to make comparing easier ;)